### ISSUE TITLE
vala-lint support

### ISSUE THREAD

**Issue by ondratu** [arrufat/vala.vim#11]
That could be amazing, if vala.vim could support vala-lint checker from https://github.com/vala-lang/vala-lint

**Comment by arrufat** [comment]
Yes, that would be cool, however, I think once the GVls (Vala language server) is completed, that might be a better option.

**Comment by atsuya** [comment]
FYI, I added support for vala-lint for ALE, if you are using ALE: https://github.com/atsuya/ale/tree/feature/support-vala-lint .
